systemtest/sharedClasses/playlist.xml

```diff
@@ -24,12 +24,15 @@
 		<variations>
 			<variation>NoOptions</variation>
 		</variations>
-		<command>perl $(SYSTEMTEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl \
+		<command>export platform=`uname`; \
+	if [ $$platform == $(Q)Darwin$(Q) ] ; then export original_umask_val=`umask` $(AND_IF_SUCCESS) umask 0002 $(AND_IF_SUCCESS) umask -p; fi; \
```

Comment:
`PLATFORM` is a special variable that TKG uses. Please do not reset it. Can we just use `PLATFORM` from TKG? I think it make sense to put the code into a systemtest.mk https://github.com/AdoptOpenJDK/openjdk-tests/blob/7331e67bbca0c723af31398afa8cb060ff8a8713/systemtest/systemtest.mk

Reply:
Spoke to Lan offline. 

- Update PR to use $(PLATFORM) variable, instead of using `uname`. 
- Decided to leave the code in playlist.xml since it only affects one test. 

Tested latest changes: 
OSX: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1908/consoleFull
Windows: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1909/console